.github/workflows/assets.yml

```diff
@@ -106,7 +106,59 @@ jobs:
           asset_name: ${{ steps.wheel.outputs.wheel }}
           asset_content_type: application/zip
 
-  windows:
+  manylinux-arm64:
+    name: PyPI wheels for Manylinux
+    runs-on:
+      - "self-hosted"
+      - "ARM64"
+      - "Linux"
+    env:
+      BISON_VERSION: "3.0.4-2.el7"
+      SWIG_VERSION: "3.0.12-17.el7"
+    strategy:
+      matrix:
+        manylinux:
+          - 2014
+        cpython_version:
+          - "cp38-cp38"
+          - "cp39-cp39"
+          - "cp310-cp310"
+          - "cp311-cp311"
+    steps:
+      - name: Checkout with submodules
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Install dependencies
+        run: |
+          sudo yum install -y flex bison swig cmake
+          /opt/python/${{ matrix.cpython_version }}/bin/python -m pip install auditwheel
+      - name: Build wheel
+        env:
+          QX_CPU_COMPATIBILITY_MODE: OFF
```

Comment:
I am not sure this is going to work because the package would then potentially fail to run on ARM cpus that are not exactly identical to the one on this device.

I think QX_CPU_COMPATIBILITY_MODE=OFF should be used only when building the package on the machine that runs it later on. So not in the CI, IMO. But I could add it to the tests workflows, since those are not published.

Reply:
Done